include/count_passenger.h

```diff
@@ -0,0 +1,6 @@
+#ifndef __COUNT_PASSENGER_H__
```

Comment:
we need to add a doxygen file comment like this. Somethin like this
/**@file count_passengers.h 
 *@brief contains declarations for count_passengers.h
*/